manager/api/micro/pom.xml

```diff
@@ -238,5 +239,31 @@
         <filtering>true</filtering>
       </resource>
     </resources>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+          <Import-Package>
+            javax.servlet;version="[2.6,4)",
+            javax.servlet.http;version="[2.6,4)",
+            org.eclipse.jetty.*;version="[8.0,10)",
```

Comment:
@EricWittmann @cmoulliard is there standard to way to track those version numbers so that if we alter ours (in the main apiman pom) that these will be reflected in the OSGi imports?